Vungle/build.gradle

```diff
@@ -9,6 +10,6 @@ dependencies {
     if (sdkSrcDependency) {
         implementation project(':vungle-android-sdk:publisher-sdk-android')
     } else {
-        implementation "com.github.vungle:vungle-android-sdk:6.7.0"
+        implementation "com.github.vungle:vungle-android-sdk:${sdkVersion}"
```

Comment:
@HuFangshuai minor question, is it safe to use jitpack here?

Reply:
> @HuFangshuai minor question, is it safe to use jitpack here?

like https://github.com/mopub/mopub-android-mediation/compare/master...Vungle:propagate-67x-to-develop#diff-845e0094d607326504a93f892bdad1b6R13 , are we making a PR to upstream from develop or another branch?

Reply:
this should be fine, the upstream PR will be in Draft stage and not merged for a bit until we release SDK on bintray/jcenter, then we can always go and make changes to our branch updating SDK dependency